docker-run.sh

```diff
@@ -391,7 +391,7 @@ echo "#---"
 if [ "$RUN_MODE" = "DEV" ]
 then
 	# Build the DEV mode container
-	docker build --target alpine-ruby-node -t dev-to:dev . || exit $?
+	docker build -t dev-to:dev . || exit $?
```

Comment:
I think we can omit the pipe? The script should fail automatically should a command not exit with 0.

Reply:
I wasn't sure either so I checked just now. In case of an error it actually just prints the `docker build` error message and moves on in the script. The `|| exit $?` apparently is needed to stop continuation.